api/server/router/experimental.go

```diff
@@ -39,8 +38,16 @@ func (r *experimentalRoute) Disable() {
 	r.handler = experimentalHandler
 }
 
+type notImplementedError struct{}
+
+func (notImplementedError) Error() string {
+	return errExperimentalFeature.Error()
```

Comment:
If this is the only place `errExperimentalFeature` is used, might as well make it a string.